shard-operator-parent/shard-operator-v1/src/main/java/com/redhat/service/smartevents/shard/operator/v1/resources/istio/authorizationpolicy/AuthorizationPolicy.java

```diff
@@ -1,4 +1,4 @@
-package com.redhat.service.smartevents.shard.operator.resources.istio.authorizationpolicy;
+package com.redhat.service.smartevents.shard.operator.v1.resources.istio.authorizationpolicy;
```

Comment:
`authorizationpolicy` module could be moved to the core

Reply:
Done. I've moved `knative` and `istio` to `-core`.